platform/podjs/rollup.config.js

```diff
@@ -15,25 +32,15 @@ export default [
                 },
             },
         ],
-        plugins: [
-            json(),
-            resolve(),
-            commonjs(),
-            sucrase({
-                exclude: ["node_modules/**"],
-                transforms: ["typescript"],
-            }),
-        ],
-        context: "window",
+        ...common,
         external: ["chai"],
-        onwarn: (warning) => {
-            if (
-                warning.code === "CIRCULAR_DEPENDENCY" &&
-                warning.cycle[0].match(/fast-check/)
-            ) {
-                return;
-            }
-        },
+        onwarn: (warning, warn) =>
```

Comment:
Why not putting this inside the `common` object, but duplicate it instead?

Reply:
Ah, I see for one build target the issue is with `fast-check` with the other with `fast-check` and `chai`. I suppose we could just ignore the warning for both libraries in both build targets in favor of a simpler configuration, what do you think?

Reply:
Yea, that would work.